sky/tools/create_macos_framework.py

```diff
@@ -94,15 +94,21 @@ def main():
   ])
   process_framework(dst, args, fat_framework, fat_framework_binary)
 
-  # Set fat_framework file attributes to read and execute for all.
+  # Add group and other readability to all files.
   subprocess.check_call([
-      'chmod', '-R', '644',
+      'chmod', '-R', 'og+r',
       os.path.join(fat_framework, 'Versions')
   ])
-  subprocess.check_call([
-      'chmod', '-R', '755',
-      os.path.join(fat_framework, 'Versions', 'A', 'FlutterMacOS')
-  ])
+  # Find all the files below the target dir with owner execute permission
+  find_subprocess = subprocess.Popen([
+      "find", fat_framework, "-perm", "-100", "-print0"
```

Comment:
Actually, I guess to be consistent with the earlier version of the code, it should be `os.path.join(fat_framework, 'Versions')` instead of just `fat_framework`, unless it's OK that it do this for all the files.

If it is OK, then (if it's OK) you could also change the one on line 100 to be just `fat_framework` too.

Just trying to make them consistent.  As it is now, it will add read permission to everything under `Versions`, but execute permission to everything that also has owner execute permission under the parent directory, `FlutterMacOS.framework`, which might be a larger group.